src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
+	 *   composition: >
+	 *     Popovers consist of a short title and text.
+	 *   effect: >
+	 *     Popovers are typically shown by clicking or hovering over a component. The position
+	 *     of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears at the top, right, bottom or left of the triggerer component.
+	 * context: >
+	 * rules:
+	 *   usage:
+	 *     1: Popovers MUST contain a title and a text.
+	 *     2: The text MUST be plain text (no HTML) and no longer than a paragraph.
```

Comment:
Those rule are insufficient proposal:
**Style**

- If Popovers do not point to content in the Main Header, they MUST be displayed behind the Main Header if the page is scrolled upwards.
- Popovers MUST always relate to the trigger element or the respective Input Element by a little pointer.

**Wording:** 

- Popovers SHOULD contain a title which names the interaction to be performed in the popover.
- Interaction:
- A Popover MUST only be displayed if the trigger element is clicked. This behaviour is different from tooltips that appear on hovering.
- Popovers MAY contain shortcuts to other services.

**Accessibility:**

- There MUST be a way to open the Popover by only using the keyboard. Exceptions are acceptable for input that can be entered by only using the keyboard without activating the Popover. In this case the Input Element MUST indicate the format of the input by setting the placeholder of the Input Element accordingly.
- The focus MUST be inside the Popover, once it is open if it contains at least one interactive item. Otherwise to focus MUST remain on the trigger.
- The focus MUST NOT leave the Popover for as long as it is open. 
- There MUST be a way to reach every control in the Popover by only using the keyboard. Exception are acceptable for controls that are only there for convenience (e.g. elements used to reduce the number of clicks to achieve a certain input such as the switch form the month-view to the year-view in the Date/Time Picker).
- The Popover MUST be closable by pressing the ESC key.
- Once the Popover is closed, the focus MUST return to the element triggering the opening of the Popover or the element being clicked if closed by click.